sass/elements/_elements.scss

```diff
@@ -78,7 +78,7 @@ img {
 
 	&:after {
```

Comment:
@audrasjb When you get a chance, please remove both the `&:before` and `&:after` pseudo elements entirely and we can get this merged in. Thanks! 👍